app/javascript/dashboard/components/layout/sidebarComponents/SecondaryNavItem.vue

```diff
@@ -161,8 +145,8 @@ export default {
         this.menuItem.toStateName === 'settings_applications'
       );
     },
-    isArticlesView() {
-      return this.$store.state.route.name === this.menuItem.toStateName;
+    isCurrentRoute() {
```

Comment:
I tested it locally and LGTM.

@nithindavid  Active colour for the count is not working.

**Before**
<img width="196" alt="image" src="https://user-images.githubusercontent.com/64252451/197112262-0a25c8aa-7ba5-43db-8a9f-7bdbcd8b0b76.png">

**After**
<img width="196" alt="image" src="https://user-images.githubusercontent.com/64252451/197112284-2cddc6be-ed88-40da-8fb4-a234f96abc09.png">